### DISCUSSION TITLE
how to change hold alt on Vomnibar to hold meta (windows key)

### DISCUSSION THREAD

**Discussion by RFDAJE** [gdh1995/vimium-c#1228] [Q&A]
holding ALT when Vomnibar is activated, we have a list of numbers in vertical order showing up, since ALT is used by my window manager, is it possible to replace ALT with META windows key? and if yes, how? Thanks

**Comment by gdh1995** [comment]
```
mapKey <c-d:o> <a-alt>
```

And `c-d` means Ctrl+D

**Reply by RFDAJE** [reply]
this mapping does make the numbered list showed up on Vomnibar, but ctrl+1,2,3 is already default keyboard shortcut to switch tab.

**Comment by gdh1995** [comment]
As for META, you may try `mapKey <m-modifier:o> <a-alt>`. Sorry but I can not test metaKey on my Windows 11.

**Reply by RFDAJE** [reply]
tried the mapping on linux chrome, but after Vomnibar activated with opened tabs, click meta/windows key nothing happened, the numbered list does not appear.

**Reply by RFDAJE** [reply]
`mapKey <c-modifier:o> <a-alt>` make ctrl to act as alt, so far this is a workaround.

**Comment by RFDAJE** [comment]
`mapKey <modifier:o> <a-alt>` does bring up the number list. but meta+1,2,3 won't switch tab, only as normal text input

**Comment by RFDAJE** [comment]
how to disable `<a-alt>` on Vomnibar? I tried the following, but it does not work.
```
unmap <a-alt:o> 
or 
unmap <a-alt>
> Error: Unmap: "<a-alt:o>" has not been mapped.
```
